app/models/invite_redeemer.rb

```diff
@@ -109,7 +115,9 @@ def send_welcome_message
   end
 
   def approve_account_if_needed
-    invited_user.approve(invite.invited_by_id, false)
+    if get_existing_user
```

Comment:
Somewhere there is a password_reset test  that checks if the user is approved or not and since its a password reset the user already exists and doesn't go into `create_user_from_invite`.

I'll have to check again but I added the `get_existing_user` check because if its an invite it should not go to the User model to approve the user.